exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -390,4 +391,15 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  /**
+   * Option whose value is a long value representing the number of bits required for computing ndv (using HLL)
+   */
+  LongValidator NDV_MEMORY_LIMIT = new PositiveLongValidator("exec.statistics.ndv_memory_limit", 30, 20);
```

Comment:
These validators are used for user-settable system or session options. The user can choose the NDV bit count? What happens if I do one run with one value, the next run with a different value? Do we handle a mix of lengths? If we don't, should this e a runtime option?